cooltools/cli/coverage.py

```diff
@@ -49,6 +49,14 @@
     is_flag=True,
     default=False,
 )
+@click.option(
+    "--clr_weight_name",
+    help="Name of the weight column. Specify if wanting to calculate"
+    " balanced coverage. In this case, cis counts will not be stored.",
```

Comment:
How about this?
Name of the weight column. Specify to calculate coverage of a balanced cooler. If specified, then the cis and tot values for balanced coverage are stored rather than raw coverage.

Reply:
Okay I've changed the help to be more explicit about this for store=True. Let me know